package.json

```diff
@@ -17,7 +20,8 @@
   "devDependencies": {
     "eslint": "^2.7.0",
     "eslint-config-loopback": "^1.0.0",
-    "loopback": "^3.0.0",
+    "loopback": "^2.36.0",
+    "loopback-datasource-juggler": "^2.53.0",
```

Comment:
Remove juggler since it was not here originally (and not being used in this module at all).